MAINTAINERS.yml

```diff
@@ -3633,8 +3633,9 @@ TI K3 Platforms:
     - vaishnavachath
   collaborators:
     - gramsay0
+    - dnltz
```

Comment:
@dnltz are you a Zephyr "contributor"?
(I just discovered this doesn't do anything otherwise)

Reply:
I would say yes. According to https://docs.zephyrproject.org/latest/project/project_roles.html, it's anyone who contributes which I already did before.

Reply:
I thought that too, but you have to join the Zephyr github project first.
You can apply by making a "Contributor Nomination" issue:
https://github.com/zephyrproject-rtos/zephyr/issues/new/choose

Reply:
interesting ... thanks! Opened an issue to nominate myself